packages/devtools_app/lib/src/shared/config_specific/post_message/_post_message_web.dart

```diff
@@ -2,14 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be found
 // in the LICENSE file.
 
-// TODO(https://github.com/flutter/devtools/issues/6606): remove this import.
-// This is the final dart:html import in DevTools. In order to remove it, we
-// need to bump the `package:web` version in DevTools to > 0.3.1, but we are
-// blocked on `package:web` rolling into the Flutter SDK.
-import 'dart:html' as html;
 import 'dart:js_interop';
 
-import 'package:web/helpers.dart';
+import 'package:web/helpers.dart' as html;
```

Comment:
why prefix this with html?

Reply:
I think it's just keeping consistent with the old syntax, but since we don't need `dart:html` anymore, there are no conflicts, and we can just remove the prefix.